### DISCUSSION TITLE
Verified teacher but no access to 'Coplot'. Any ideas?

### DISCUSSION THREAD

**Discussion by mbehrisch** [community/community#109032] [GitHub Education]
### Select Topic Area

Question

### Body

Hi, I am verified as a teacher. 
![Screenshot - 16-2-2024 , 14_43_14](https://github.com/community/community/assets/310635/8d4eb8bd-31f7-4ade-9413-26ce937fab39)

But my Github Copilot page (https://github.com/settings/copilot) does not allow me to enable Copilot. 
The only options are 1) "Buy Copilot"
![Screenshot - 16-2-2024 , 14_46_59](https://github.com/community/community/assets/310635/57d3c26a-039c-42c0-bc83-f75a3f5c5057)

2) and get from an organization, but my University is not showing up in this list only in the "Enterprises". However, option 2 points me to the education program. :)

NB: https://github.com/github-copilot/signup states that: 
![Screenshot - 16-2-2024 , 14_51_02](https://github.com/community/community/assets/310635/e7236a4c-125a-4543-bcd2-5c46f900dc24)
Is this a related or unrelated issue?

Any idea how this can be fixed?
Michael Behrisch

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation

Acces the [Github Support](https://support.github.com/)

Click on My Ticket :

![github04](https://github.com/community/community/assets/72712095/096b3477-cee6-4a19-9f1b-6fb2219b51a1)

and open a New Ticket:

![github05](https://github.com/community/community/assets/72712095/7413fc55-875a-498c-93eb-5c54d5ffc8d6)


God bless you!
